src/Components/Home/Homepage.js

```diff
@@ -0,0 +1,74 @@
+import { React,useEffect,useState } from "react";
+import Footer from "../Footer/Footer";
+import Navbar from "../Navbar/Navbar";
```

Comment:
Have an index.js file in the components folder, import all the components over there, and then export from that file, so when you import, then you no need to import different components in a separate line.
e.g import {Footer,  Navbar} from './components/'